### DISCUSSION TITLE
Using ImageMagick library to take snapshot of a image and store the snapshot in webp format

### DISCUSSION THREAD

**Discussion by cparkhi16** [ImageMagick/ImageMagick#6406] [Help]
### ImageMagick version

7.1.1-11

### Operating system, version and so on

Linux

### Description

Is it possible to take snapshot of a image of any file format and store it in webp format ?
If yes, can I get sample examples for it or a documentation link for it ?

**Comment by snibgo** [comment]
I'm not sure what you mean by "take snapshot of a image of any file format".

IM can read an image file of almost any format, and write the image to WEBP file format. For example:
```
magick in.png out.webp
```
